setup.py

```diff
@@ -1,5 +1,9 @@
 import os
+import sys
 from setuptools import setup
+
+assert sys.version_info >= (3, 6), "crypt4gh requires at least python3.6"
```

Comment:
It doesn't hurt to keep it in the `__main__` section, right?

Just a question: should we put it inside the other modules too, so that `import` also catches it?